File: TravelAgency.Domain/Models/Passenger.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TravelAgency.Domain.Models
{
    public class Passenger : BaseEntity
    {
        [Required]
        [MaxLength(50)]
        public string FirstName { get; set; } = string.Empty;

        [Required]
        [MaxLength(50)]
        public string LastName { get; set; } = string.Empty;

        [Required]
        [MaxLength(100)]
        public string PassportNumber { get; set; } = string.Empty;

        [Required]
        public DateTime PassportExpirationDate { get; set; }

        [Required]
        public DateTime BirthDate { get; set; }

        [MaxLength(50)]
        public string? Email { get; set; }

        [MaxLength(30)]
        public string? PhoneNumber { get; set; }

        [MaxLength(50)]
        public string? Address { get; set; }

        [MaxLength(250)]
        public string? Comment { get; set; }

        [Required]
        public int ContractId { get; set; }

        [ForeignKey("ContractId")]
        public Contract Contract { get; set; }
    }
}

File: TravelAgency.Frontend/travelagency/src/app/shared/models/passenger.ts
export interface PassengerCreateModel {
    firstName: string;
    lastName: string;
    passportNumber: string;
    passportExpirationDate: string;
    birthDate: string;
    email?: string;
    phoneNumber?: string;
    address?: string;
    comment?: string;
}

export interface PassengerDetailsModel {
    firstName: string;
    lastName: string;
    passportNumber: string;
    passportExpirationDate: Date;
    birthDate: Date;
    email?: string;
    phoneNumber?: string;
    address?: string;
    comment?: string;
}